pom.xml

```diff
@@ -146,7 +146,7 @@
 		<dependency>
 			<groupId>org.venice.piazza</groupId>
 			<artifactId>pz-jobcommon</artifactId>
-			<version>2-0-1-s7-4-ge60730f</version>
+			<version>JWT</version>
```

Comment:
This will be updated when JobCommon is peer reviewed and merged with a new version.